app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -0,0 +1,383 @@
+# frozen_string_literal: true
+
+# OVERRIDE: Override the file from Hyrax to modify methods usage
```

Comment:
I can't tell what part you modified from this comment. In the future please put the `OVERRIDE` comment near the code that changed.

Reply:
I will keep that in mind for next PR to show where the changes happen.